tests/pipelines/stable_diffusion/test_stable_diffusion_adapter.py

```diff
@@ -57,23 +58,24 @@ class AdapterTests:
     def get_dummy_components(self, adapter_type):
         torch.manual_seed(0)
         unet = UNet2DConditionModel(
-            block_out_channels=(32, 64),
+            block_out_channels=(32, 32, 32, 64),
```

Comment:
Could I get an explanation of why the default changes are being changed here? IMO, the default tests should be unchanged and if needed, new tests should be added.

Reply:
Reverted changes to existing tests in [13c9f79](https://github.com/huggingface/diffusers/pull/5435/commits/13c9f79e87bd875ca0e783d01c24813b03e96b24) and added new test setup in [57d55c7](https://github.com/huggingface/diffusers/pull/5435/commits/57d55c7bfe1e8906c2933c393af1d42b680f17d7).

(There is no longer any diff in the expected test results. Which hopefully gives more confidence that this PR does not introduce any regressions.)

Reply:
Thank you!